easybuild/easyblocks/craytoolchain.py

```diff
@@ -0,0 +1,84 @@
+##
```

Comment:
This could have side effects on other systems, can you please comment why you need to override the craytoolchain easyblock?

Reply:
This change was addressing an issue with Modules versions 4.0 and 4.1: 
```
On versions 4.0 and 4.1, unloading an unexistent modulefile generates an Unable to locate modulefile error. Starting with version 4.2, unloading a module only looks at loaded module list and does not trigger a modulefile search. So starting version 4.2 the same behavior than Modules compatibility version is obtained.
```
See section `unload` in https://modules.readthedocs.io/en/latest/diff_v3_v4.html

Since this issue was affecting MCH workflow, we have later introduced the compatibility mode of Modules, which reverts to Modules 3.2.1:
```
tsa-ln002:~$ which modulecmd
/cm/local/apps/environment-modules/4.0.0/bin/modulecmd
lucamar@tsa-ln002:~$ modulecmd -V | grep -w VERSION
VERSION=3.2.11 
```
Therefore, this custom easyblock is currently not necessary anymore.